.github/dependabot.yml

```diff
@@ -20,6 +20,8 @@ updates:
       - dependency-name: mxschmitt/action-tmate
       - dependency-name: step-security/harden-runner
       # Managed by cisagov/skeleton-ansible-role
+      # - dependency-name: docker/setup-buildx-action
+      # - dependency-name: docker/setup-qemu-action
```

Comment:
Don't forget to uncomment these guys!

Reply:
Thanks for the reminder! 🤦   Done in https://github.com/cisagov/ansible-role-okta-asa-server-agent/pull/9/commits/d06165080473420bdff302984905cfd993a93da3.